tools/syz-testbed/testbed.go

```diff
@@ -117,6 +107,28 @@ func main() {
 	ctx.Loop(shutdown)
 }
 
+func (ctx *TestbedContext) MakeMgrConfig(base, patch json.RawMessage) *mgrconfig.Config {
+	var mergedConfig []byte
```

Comment:
Can't this be just:
```
	mergedConfig, err := config.MergeJSONData(base, patch)
	if err != nil {
		tool.Failf("failed to apply a patch to the base manager config: %s", err)
	}
```
?

Reply:
Technically, "" (empty string) is an invalid JSON and consequently `json.Unmarshal` fails on that.

Reply:
But these are not strings, these are json.RawMessage ([]byte).

Reply:
It looks reasonable to be able to merge empty/nil config. I assume if we use this functionality anywhere else for similar purpose, any user will need to duplicate this logic. So better to move into MergeJSONDatae

Reply:
Done